packages/provider/src/transports/wallet-request-handler.ts

```diff
@@ -771,13 +771,18 @@ export class WalletRequestHandler implements ExternalProvider, JsonRpcHandler, P
     this.events.emit('disconnect')
   }
 
+  notifyChainChanged(chainId: number, origin: string) {
```

Comment:
`origin` is ok to be mandatory here, right? I think that's fine, just wanted to check with you.